### ISSUE TITLE
Tooltip.js + tether.js throwing error: tether.js:113 Uncaught TypeError: Failed to execute 'removeChild' on 'Node': parameter 1 is not of type 'Node'.

### ISSUE THREAD

**Issue by crw** [twbs/bootstrap#19843]
Bootstrap v4.0.0-alpha.2
Tether 1.3.1
Chrome Version 50.0.2661.94 (64-bit) 
OS X 10.11.4 (15E65)

I am attaching Tooltips to various elements in the page. For any one of these tooltips, the first time the tooltip appears correctly; subsequent attempts to make the same tooltip appear result in the above error. 

Apparently, when Tooltip.js calls "tether.destroy()", it removes an element that will not be reconstructed on subsequent attempts to open the tooltip. I have narrowed it down to the variable "zeroElement" in tether.js no longer existing after calling tether.destroy(). 

Commenting out the internals of this method in Tooltip.js fixes the issue:

```javascript
      key: 'cleanupTether',
      value: function cleanupTether() {
        if (this._tether) {
          this._tether.destroy();
        }
      }
```

**Update** A workaround is to revert to tether.js version 1.1.0.

**Comment by RyanZim** [comment]
X-Ref:  #19790

**Comment by crw** [comment]
JSFiddle that demonstrates the issue:

https://jsfiddle.net/r3qk0aud/

Note that this is an issue as of Tether 1.3.1; Tether 1.1.0 which is used by the Bootcamp docs site does not exhibit this issue.

**Comment by twbs-lmvtfy** [comment]
Hi @crw!

You appear to have posted a live example (https://fiddle.jshell.net/r3qk0aud/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 31, column 7 thru column 53: No space between attributes.
* line 31, column 7 thru column 60: A slash was not immediately followed by ```` > ````.
* line 31, column 7 thru column 61: A slash was not immediately followed by ```` > ````.
* line 31, column 7 thru column 85: A slash was not immediately followed by ```` > ````.
* line 31, column 7 thru column 95: A slash was not immediately followed by ```` > ````.
* line 31, column 7 thru column 109: A slash was not immediately followed by ```` > ````.
* line 31, column 7 thru column 113: A slash was not immediately followed by ```` > ````.
* line 31, column 7 thru column 130: Quote ```` " ```` in attribute name. Probable cause: Matching quote missing somewhere earlier.
* line 31, column 7 thru column 131: Bad value ````  ```` for attribute ```` href ```` on element ```` link ````: Must be non-empty.
* line 31, column 7 thru column 131: Attribute ```` https: ```` not allowed on element ```` link ```` at this point.
* line 31, column 7 thru column 131: Attribute ```` maxcdn.bootstrapcdn.com ```` not allowed on element ```` link ```` at this point.
* line 31, column 7 thru column 131: Attribute ```` bootstrap ```` not allowed on element ```` link ```` at this point.
* line 31, column 7 thru column 131: Attribute ```` 4.0.0-alpha.2 ```` not allowed on element ```` link ```` at this point.
* line 31, column 7 thru column 131: Attribute ```` css ```` not allowed on element ```` link ```` at this point.
* line 31, column 7 thru column 131: Attribute ```` bootstrap.min.css" ```` not allowed on element ```` link ```` at this point.
* line 71, column 3 thru column 8: Start tag ```` body ```` seen but an element of the same type was already open.
* line 98, column 1 thru column 7: Saw an end tag after ```` body ```` had been closed.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by crw** [comment]
Updated, although I do not know if it will fix the validation errors. FWIW, I do not intend to fix the validation errors. 

https://jsfiddle.net/r3qk0aud/1/

**Comment by crw** [comment]
I can confirm that reverting to tether.js v1.1.0 solves this problem for my application. 

Yay?

**Comment by crw** [comment]
Opened a ticket on the tether.js side of things just to give them a heads-up, since I am not sure where the bug belongs: https://github.com/HubSpot/tether/issues/173

**Comment by crw** [comment]
This issue is closed, fixed by the release of Tether 1.3.2. Thanks!
